tot/models/models_simple.py

```diff
@@ -193,6 +235,10 @@ def predict(self, df: pd.DataFrame, df_historic: pd.DataFrame = None):
                 dataframe containing column ``ds``, ``y``, and optionally ``ID`` with data
             df_historic : pd.DataFrame
                 dataframe containing column ``ds``, ``y``, and optionally ``ID`` with historic data
+            received_ID_col : bool
```

Comment:
would make readability better for me personally
```suggestion
            received_id_column : bool
```